lib/controllers/docs_controller.ex

```diff
@@ -69,6 +70,29 @@ defmodule ApplicationRunner.DocsController do
     end
   end
 
+  def create(
+        conn,
+        %{"coll" => coll},
+        doc,
+        %{environment: env, transaction_id: transaction_id},
+        replace_params
+      ) do
+    with filtered_doc <- Map.delete(doc, "_id"),
```

Comment:
It seems do be almost the same code. Isn't it better to manage it in the same function ? I don't know the good practices in Elixir ^^